### PR TITLE
Reduce Bundle Size pt 1

### PR DESCRIPTION
**Pull request by daniel-heppner-ibigroup**
This addresses the following bundle size problems:
- Fix styled-icons code splitting. ~~(Blocked by https://github.com/opentripplanner/otp-ui/pull/456)~~
- Fix unneeded import in mailables for pdfkit
- Removes stylized map completely, along with d3.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -68,8 +68,6 @@
     "copy-to-clipboard": "^3.0.8",
     "country-flag-icons": "^1.4.19",
```
Comment (miles-grant-ibigroup) [inline package.json:69]: While we're removing things, I don't think this is used anymore

File: package.json [hunk 2]
```diff
@@ -68,8 +68,6 @@
...
     "currency-formatter": "^1.4.2",
```
Comment (miles-grant-ibigroup) [inline package.json:70]: Same for this

**Review by miles-grant-ibigroup — APPROVED** [review]
Seems like the Percy tests are facing the am/pm bug again. I think they really are affected by the time they're run... I'll try re-running them tomorrow morning and we can see if that changes thigns

File: package.json (hunk 2 above)
Comment (daniel-heppner-ibigroup) [reply package.json:70]: It's used in tnc-leg. It seems like we might not be using tnc-leg anymore though. Can it be removed?

File: package.json (hunk 1 above)
Comment (daniel-heppner-ibigroup) [reply package.json]: Shouldn't affect bundle size, but it's been removed.

**Pull request merged**